### PR TITLE
fix: rework environment mocks, organize fixtures and mocks

### PR DESCRIPTION
**Pull request by dylandepass**
Try and make environment mocking more robust, as we add more code this may need to be extended.

Cleaned up the test collateral, moved mocks into mocks and fixtures into fixtures.

### CODE REVIEW

**Comment by dylandepass** [comment]
@rofe The drop in coverage is caused by me stubbing some plugins (where they weren't stubbed before).

As we add test coverage for plugins these lines will get covered again.

File: test/wtr/mocks/environment.js [hunk 1]
```diff
@@ -10,107 +10,223 @@
...
+};
+
+/**
+ * Mock helix environments
+ * @enum {string}
+ */
+// eslint-disable-next-line no-unused-vars
+const HelixMockEnvironments = {
+  INNER: 'inner',
+  OUTER: 'outer',
```
Comment (rofe) [inline test/wtr/mocks/environment.js:48]: let's use 
```suggestion
  PREVIEW: 'preview',
  LIVE: 'live',
```

File: test/wtr/mocks/environment.js [hunk 2]
```diff
@@ -10,107 +10,223 @@
...
+ * @param {HelixMockContentType} contentType The content type for the default environment location
+ * @param {HelixSecondLevelDomains} sld The second level domain to use in the environment
+ * @returns {Object | undefined}
+ */
+export function getDefaultHelixEnviromentLocations(contentType, sld) {
+  switch (contentType) {
+    case HelixMockContentType.DOC:
+      return {
+        inner: `https://main--aem-boilerplate--adobe.${sld}.page`,
+        outer: `https://main--aem-boilerplate--adobe.${sld}.live`,
```
Comment (rofe) [inline test/wtr/mocks/environment.js:97]: and here:
```suggestion
        preview: `https://main--aem-boilerplate--adobe.${sld}.page`,
        live: `https://main--aem-boilerplate--adobe.${sld}.live`,
```

**Review by rofe — APPROVED** [review]
LGTM

**Pull request merged**